src/modules/rlm_ldap/rlm_ldap.h

```diff
@@ -150,6 +150,18 @@ extern HIDDEN fr_dict_attr_t const *attr_user_name;
 /*
  *	user.c - User lookup functions
  */
+#define rlm_ldap_find_user_cached(_dn, _request) \
```

Comment:
just static inline it...  Macros are fine if you want to manipulate other variables in scope directly, or implement foreach loops, but they have no advantages over static inlines, and are harder to debug in a compiler.